src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -95,6 +108,27 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
             ])
+            ->add('show_catalog_price_rules', IconButtonType::class, [
+                'icon' => 'visibility',
+                'label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
```

Comment:
On the mockups this is the same button for cart rules and catalog rules : 
![image](https://user-images.githubusercontent.com/70583503/175258364-f4bc45ef-aa20-43bb-baf0-49ce6b30850b.png)

Reply:
Indeed but for now we only implemented the catalog rules, not the cart rules which is why the wording needs to change a little And by default the list is hidden so the initial wording is `Show ...` but maybe it should be `Show catalog rules` ? But it's not really consistent with the naming of the menu and list:
![Capture d’écran 2022-06-27 à 10 58 04](https://user-images.githubusercontent.com/13801017/175901938-e678e65a-cce4-4997-b190-558204ff009c.png)

Reply:
Ok, thank you, in that case, "Show catalog price rules" is fine. You're right we should stay consistent.